### ISSUE TITLE
Builder not placing double brick slabs

### ISSUE THREAD

**Issue by Curse187** [McJty/RFTools#1547]
Minecraft 1.10.2
rftools-1.1x-7.11

The Builder make some wired stuff. it can't place double brick slabs, doors, torches...

**Comment by McJty** [comment]
First that's an old version of RFTools.
Second you need to give more information. Exaclty how are you trying to place this?
Also some things are never going to be placable with the builder. That includes doors (as they are two high and that's complicated to handle) and double slabs (as that requires two blocks in the same blockspace)

**Comment by Curse187** [comment]
I hope, this explains it more.

![bild1](https://user-images.githubusercontent.com/26227796/34561018-7a5dfdde-f148-11e7-9703-cf998f9dac27.jpg)
![bild2](https://user-images.githubusercontent.com/26227796/34561033-80d76664-f148-11e7-9534-ef46ba8abe49.jpg)
![bild3](https://user-images.githubusercontent.com/26227796/34561034-80f9bbf6-f148-11e7-9f67-fb785bc795f7.jpg)
![bild4](https://user-images.githubusercontent.com/26227796/34561036-8119c7c0-f148-11e7-9976-9517affd892d.jpg)

**Comment by McJty** [comment]
Yes, the double slabs are problematic. I don't know how to solve that

**Comment by Curse187** [comment]
Maybe switch them with the solid compartment?

**Comment by McJty** [comment]
The two-slab doesn't exist as an item so you can't put it in the chest. Also the Builder cannot handle the fact that two items in a chest are needed to make a single block in the world and it is very complicated to implement that

**Comment by Curse187** [comment]
Is it possible for the Builder to recognize if there is a single or a double slab?

If this is possible, can you exchange the slab with a full block?

**Comment by McJty** [comment]
Then you would have an infinite slab generator: one builder to place a single slab and place it as double. Another builder to harvest the two slabs

**Comment by Curse187** [comment]
sorry, i think you get me wrong.

Is it possible for the Builder to analyze if there is a single or a double slab?

If this is possible, can you replace the double slab with a full block? 
maybe you can do so with other blocks they are cause problems. ignore them or replace them.


i think the problem with the doors and the torches can be easy fix.
--> simply change the direction the builder work to bottom to top.

**Comment by McJty** [comment]
Replace the double slab with what full block? I don't understand your suggestion

The builder already works from bottom to top

**Comment by Curse187** [comment]
With this:

![bild 5](https://user-images.githubusercontent.com/26227796/34576800-66ccf144-f17f-11e7-9e59-35677c2bdd17.png)

**Comment by ouroborus** [comment]
IMHO, double slabs are a vanilla mechanic and should be handled properly by any mod that manipulates blocks.
